mobile/src/context/BookContext.tsx

```diff
@@ -20,13 +20,208 @@ export const BookProvider: React.FC = ({ children }) => {
   const [state, dispatch] = useReducer(reducer, initialState);
   const client = useContext(APIContext);
 
-  // loads books. If error, logs erorr and returns a loading indicator of true
+  // loads books. If error, logs error and returns a loading indicator of true
   function fetchBooks(): void {
     dispatch({ type: 'API_CALL_STARTED' });
     client.getBooks().then((res) => {
       dispatch({ type: 'API_RETURNED', payload: res });
     }).catch((err) => {
       console.log(err);
+      const books: Book[] = [
```

Comment:
Ready to remove this? Also is there something wrong with `npm run staging`? I would think that would be an easier way to get some books without running the backend

Reply:
👍 , Nothing wrong with staging, I was just trying to test different inputs and make sure the expected books were showing up.